docs/databags.md

```diff
@@ -114,6 +114,7 @@ Docs Navigation
 [server.conf]: http://docs.splunk.com/Documentation/Splunk/latest/Admin/Serverconf
 [indexes.conf]: http://docs.splunk.com/Documentation/Splunk/latest/Admin/Indexesconf
 [user-prefs.conf]: http://docs.splunk.com/Documentation/Splunk/latest/Admin/User-prefsconf
+[limits.conf]: http://docs.splunk.com/Documentation/Splunk/latest/admin/Limitsconf
```

Comment:
Removed this line note, definitely doesn't make sense here. Updated the attributes.md with a note about the limits attributes.

95bc704